contact.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+<body>
+        <body>
+                <div class="container">
+                    <div class="headinfo">
+                            <div class="head left">
+                                <a href="./index.html"><img class="logo" src="img/lambda-black.png" alt="Great Idea! Company logo." /></a></br>
+                            </div>
+                            <div class="head right">
+                              <a class="head links" href="./index.html" >Home</a> 
+                              <a class="head links" href="./about.html" >About</a> 
+                              <a class="head links" href="" >Product</a> 
+                              <a class="head links" href="" >Blog</a> 
+                              <a class="head links" href="./contact.html" >Contact</a>
+                            </div>
+                    </div>
+                    <div class="head image">
+                            <img class="logo" src="img/jumbo.jpg" alt="jumbo image" /></br>
+                        </div>
+                    <div class="form">
+                            <form action="action_page.php">
```

Comment:
Haha! I see a PHP fan right there!! Nice!!
When you get to React, things will change!!🔥